src/zenml/zen_stores/schemas/run_metadata_schemas.py

```diff
@@ -129,7 +129,9 @@ def to_model(self) -> "RunMetadataResponseModel":
             value=json.loads(self.value),
             type=self.type,
             workspace=self.workspace.to_model(),
-            user=self.user.to_model() if self.user else None,
+            user=self.user.to_model(_block_recursion=True)
+            if self.user
+            else None,
```

Comment:
feels very arbitrary - why only the user, why only for this specific model?

Reply:
Would suggest we keep those as is and tackle hydration separately

Reply:
Even though this is a minimal change compared to the changes in the configuration, this has an effect on the final representation of a pipeline run. The run metadata models are repeated quite a few times already and this change just excludes the roles in the user model (which are never used). If it is alright, I would leave this change in.